egs/wsj/s5/utils/queue.pl

```diff
@@ -65,8 +65,9 @@
 my $jobname;
 my $jobstart;
 my $jobend;
-
+my $submit_flag = 0;
```

Comment:
I think you can simplify this by removing the $submit_flag and just doing if (defined $sge_job_id) {...